frontend/src/NavBar/NavBar.module.css

```diff
@@ -0,0 +1,45 @@
+.container {
+  background: hsl(0, 0%, 85%);
+  height: 72px;
+  display: flex;
+  position: fixed;
```

Comment:
i'd personally recommend rather than using fixed, instead we will ensure the page is structured like this:

```
|          |
|          |    body
|          |    90vh or so
|__________|
|__________|   nav 10vh or so
                         = 100vh, so no scroll bar
```
then no fixed needed!

Reply:
would I just write 10vh then for height and remove position and bottom?

Reply:
yep that's fine, we'll make sure the main view plays nice accordingly

Reply:
ok thanks!